templates/boot/generic-uboot-tftp-ramdisk-boot-template.jinja2

```diff
@@ -6,6 +6,10 @@
 {{ super() }}
 {% endblock %}
 {% block actions %}
+{%- if extra_kernel_args %}
+context:
```

Comment:
I think this will overwrite the context section thats written in kernel-ci-base.jinja2
Context can currently be added in the test-configs yaml per device-type, so we should probably add a "default" context that is written to all jobs.

Reply:
OK it's possible to use the `context` jinja dictionary variable instead then, and add `extra_kernel_args` to it.

Reply:
Thanks for catching this, this is fixed in the latest version by just updating the `context` jinja dictionary.